src/stackdriver-nozzle/main.go

```diff
@@ -42,15 +50,26 @@ var (
 func main() {
 	kingpin.Parse()
 
-	client := firehose.NewClient(*apiEndpoint, *username, *password, *skipSSLValidation)
+	input := firehose.NewClient(*apiEndpoint, *username, *password, *skipSSLValidation)
 
 	sdClient := stackdriver.NewClient(*projectID, *batchCount, *batchDuration)
-	n := nozzle.Nozzle{StackdriverClient: sdClient}
+	output := nozzle.Nozzle{StackdriverClient: sdClient}
+
+	filteredOutput, err := filter.New(&output, strings.Split(*eventsFilter, ","))
+	if err != nil {
+		if unknownEvent, ok := err.(*filter.UnknownEventName); ok {
+			fmt.Printf("Error: %s, possible choices: %s\n", unknownEvent.Error(), strings.Join(unknownEvent.Choices, ","))
+			os.Exit(-1)
+		} else {
+			panic(err)
+		}
```

Comment:
I was thinking about that but I couldn't come up with a good generic name to switch on and figured this is straight forward since there's only one expected specialization